tests/transparent.rs

```diff
@@ -0,0 +1,97 @@
+use snafu::prelude::*;
+use std::error::Error;
+
+#[derive(Debug, Snafu)]
+enum AlphaError {
+    AlphaDummy,
+}
+
+#[derive(Debug, Snafu)]
+enum BetaError {
+    BetaDummy { source: AlphaError },
+}
```

Comment:
Feel free to make these struct errors instead of enums if there's no reason to have variants.